projects/ngx-charts-on-fhir/src/lib/fhir-chart/fhir-chart.component.ts

```diff
@@ -26,7 +26,9 @@ export class FhirChartComponent implements OnInit {
   @Input() width: string = '600px';
   @Input() height: string = '300px';
 
-  constructor(private configService: FhirChartConfigurationService) {}
+  @ViewChild('baseChart') baseChart?:ElementRef;
```

Comment:
Is this necessary? I don't see anything in the ts code using this. If it's only used in the template, can't it just use the template reference variable directly?

Reply:
@semicolin i have tried this but getting error 
![image](https://user-images.githubusercontent.com/117890382/211490361-0f91f4ef-04c9-499d-98da-52ade5293d59.png)
Do you have any other solution?

Reply:
Actually, after seeing the change detector error below, let's try using a service instead.

Reply:
Or you could just go back to using a hard-coded id for now. Keep it simple until we actually need something more sophisticated.